### DISCUSSION TITLE
Dark Mode

### DISCUSSION THREAD

**Discussion by dev-fredericfox** [gluestack/gluestack-ui#627] [Q&A]
Hi, I was wondering how dark mode should be implemented. I see that I can pass custom dark colors with 
```
{
  _dark: {
    color: 'xyz';
  }
}
```
And in react native I can check what mode the user has set with `useColorScheme()`, but how do I tell gluestack to switch to dark mode? 

Edit: I have tried passing light or dark into colorMode={scheme} in `<GluestackUIProvider config={config.theme} colorMode={scheme}>` but nothing happens.

**Comment by iamtomffee** [comment]
```tsx
import {useColorScheme} from 'react-native';
import {COLORMODES} from '@gluestack-style/react/lib/typescript/types';

export default function App() {
  const colorMode = useColorScheme() as COLORMODES;  //dark | light

  return (
    <GluestackUIProvider config={config} colorMode={colorMode}>
      <Something />
   </GluestackUIProvider>
   );
}
```

then in the config or in components use `_dark` like this:

```tsx
<VStack
  bg="$white"
  sx={{
    _dark: {
      bg: '$backgroundDark950',
    }
  }}
>
```
